### PR TITLE
Clarify the L2 voting process

### PR DESCRIPTION
**Pull request by matthewp**

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ Certain project decisions require a vote. These include:
 
 - Governance changes: simple majority (over 50%) conducted via GitHub PR approval.
 - Core contributor membership: overwhelming majority (over 70%) conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
+- Committer contributor membership: discussion conducted via a thread with all existing L2 contributors. Thread will be deleted once a decision is made.
```
Comment (FredKSchott) [inline GOVERNANCE.md:119]: - mention simple vs. overwhelming majority
- actually, everything added here feels relevant to both membership voting, not just committer. Does it make more sense to add this to the existing bullet point as "~core~ contributor membership" and then also remove the point about private messaging for both? I'd be +1 on that change.

**Comment by FredKSchott** [comment]
LGTM! Remember that governance changes require a majority approval before merging + 3 day window.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ Steward is an additional privilege bestowed to 1 (or more) Contributors. The rol
 Certain project decisions require a vote. These include:
 
 - Governance changes: simple majority (over 50%) conducted via GitHub PR approval.
-- Core contributor membership: overwhelming majority (over 70%) conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
+- Contributor membership (L2 and L3): discussion via a temporary channel with qualified contributors for up to 3 days. overwhelming majority (over 70%) vote conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
```
Comment (natemoo-re) [inline GOVERNANCE.md:118]:
```suggestion
- Contributor membership (L2 and L3): discussion conducted via a temporary Discord channel open to qualified contributors for up to 3 days. Acceptance requires an overwhelming majority (over 70%) vote conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
```

**Review by natemoo-re — CHANGES_REQUESTED** [review]
Just cleaning up some wording

**Review by natemoo-re — APPROVED** [review]

**Comment by FredKSchott** [comment]
@drwpow waiting for your vote to merge!

**Comment by FredKSchott** [comment]
3 days have elapsed, @drwpow automatically marked as abstaining. 
@natemoo-re @matthewp @FredKSchott marked as approved. 

PR approved!

**Pull request merged**